.github/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,28 @@
+### What is this PR for?
+A few sentences describing the overall goals of the pull request's commits.
+First time? Check out the contributing guide - https://github.com/apache/incubator-yunikorn-web/blob/master/docs/how-to-contribute.md   
```

Comment:
Pls replace the contribution guide link: http://yunikorn.apache.org/community/how_to_contribute